packages/components/src/components/error-modal/error-modal-content.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import Button from '../button';
+import Icon from '../icon';
+import Text from '../text';
+import { Localize } from '@deriv/translations';
+
+type TMessageObject = {
+    message: string;
+    toString: () => string;
+};
+
+type TErrorModalContent = {
+    error_message: Array<TMessageObject | React.ReactNode>;
+};
+
+const ErrorModalContent = ({ error_message }: TErrorModalContent) => {
+    const error_description = error_message[0]?.toString();
+
+    return (
+        <div className='unhandled-error'>
+            <Icon icon='IcAccountError' size={96} />
+            <Text
+                className='da-icon-with-message__text'
+                as='p'
+                size='s'
```

Comment:
@hamza-deriv the default value for size and color are 's' and 'general'. No need to pass it.

Reply:
sure